### ISSUE TITLE
Support ClearKey Encrypted HLS

### ISSUE THREAD

**Issue by tiberiumihai** [videogular/ngx-videogular#34]
### Description
I've packaged encrypted media files into dash and hls using MP4Box using filters to generate both manifests (dash and hls) manifest.m3u8:dual.

vgDash is working great with ClearKey DRM but I'm stuck with Hls on Safari.

I'm using both:
<video ... [vgDash]="video.dashManifest"
               [vgHls]="video.hlsManifest"
               [vgDRMToken]="video.token"
               [vgDRMLicenseServer]="video.licenseServers"><track ... /></video>

I see that vgDRMToken and vgDRMLicenseServer are available only for vgDash.
Is there any other way to specify ClearKey for vgHls?

I've seen this PR (https://github.com/video-dev/hls.js/pull/2923) and I would have tried something but apparently I don't have access to hls.js instance in vgHls.

### Expected Behavior
ClearKey Encrypted Hls to work.

### Actual Behavior
Currently, the player, loads manifest.m3u8 playlist and associated segments, but can not decrypt and displays 00:00 video.

### Steps to Reproduce
1. Encrypt video and audio files
2. Generate dash and hls manifest.m3u8:dual based on encrypted media
3. Use [vgHls]="'http://localhost:<port>/manifest.m3u8'"
4. hls can not decrypt video stream. No errors in console though.

### Attachments
![HLS Supported in Safari](https://user-images.githubusercontent.com/12754525/95993171-75f65480-0e37-11eb-9faf-58cf5314979d.png)

![DASH Supported in Chrome](https://user-images.githubusercontent.com/12754525/95993361-b1911e80-0e37-11eb-964f-474176f0b918.png)

**Comment by tiberiumihai** [comment]
ClearKey support will be added into hls.js@v1.0.0
PR here: https://github.com/video-dev/hls.js/pull/2934

Maybe we can add support for hls as well.
I can help with this once the PR is merged and release in hls.js.

**Comment by IxquitilisSaid** [comment]
Yeah for sure! I subscribed to the HLS PR so we can fix it as soon as it's merged 👀

**Comment by IxquitilisSaid** [comment]
Pinging to prevent stale status
They sure do be taking their time to merge it...

**Comment by tiberiumihai** [comment]
Yes... 

Update:
For my project I've ditched clear key support in favour of AES-128, and dash in favour of fragmented hls.
Safari doesn't support ClearKey anyway, and most major browsers would work with AES-128 fragmented hls.
So, even if hls.js player is adding support for ClearKey, a ClearKey encrypted stream won't work in Safari (Web, iPhone or iPads).

**Comment by IxquitilisSaid** [comment]
Thanks, Apple. Very cool... 🙄 
Anyway... I glad you found a workaround for your project 😄 , as for the PR once it's merged I'll add support for it either way. 
Hopefully, no one will try it on Safari

**Comment by tiberiumihai** [comment]
If you need help, I can help as well, let me know. Thanks!

**Comment by IxquitilisSaid** [comment]
Aye! I'll ping you if something comes up
Thanks, once again! 😄
